src/protocol/resp/src/lib.rs

```diff
@@ -1,16 +1,21 @@
-// Copyright 2022 Twitter, Inc.
+// Copyright 2023 Pelikan Foundation LLC.
```

Comment:
I'm not sure we can change the copyright on this. I believe we can add an additional 2023 copyright, but I don't think we can remove the current Twitter one. CC @thinkingfish

Reply:
+1, we should add a new line